React/Views/RCTWebView.h

```diff
@@ -17,6 +17,7 @@
 @property (nonatomic, assign) UIEdgeInsets contentInset;
 @property (nonatomic, assign) BOOL shouldInjectAJAXHandler;
 @property (nonatomic, assign) BOOL automaticallyAdjustContentInsets;
+@property (nonatomic, assign) BOOL openLinksInExternalBrowser;
```

Comment:
@a2 What does that mean? I don't know Objective-C; does that line have a mistake?